can/interfaces/socketcan/socketcan_constants.py

```diff
@@ -81,3 +81,18 @@
 
 EXT_ACCEPTANCE_MASK_ALL_BITS = (2**29 - 1)
 MAX_29_BIT_ID = EXT_ACCEPTANCE_MASK_ALL_BITS
+
+def parseCanFilters(can_filters=None):
+  if can_filters is None:
+    # Pass all messages
+    can_filters=[{
+      'can_id': 0,
+      'can_mask': 0
+    }]
+  
+  can_filter_fmt = "={}I".format(2 * len(can_filters))
+  filter_data = []
+  for can_filter in can_filters:
+    filter_data.append(can_filter['can_id'])
+    filter_data.append(can_filter['can_mask'])
+  return can_filter_fmt, filter_data
```

Comment:
I think it would be better just return `struct.pack(can_filter_fmt, *filter_data)` - or am I missing something?

Slightly uncomfortable adding code to a constants file, perhaps add this in a new `socketcan_common.py` module.